File: js/task2/S2_FOR_02.js
function findFirstEven(nums) {
  for (let i = 0; i < nums.length; i++) {
    if (nums[i] % 2 === 0) {
      return nums[i];
    }
  }
  return null;
}

// Tests
console.log(findFirstEven([1, 3, 7, 10, 5])); // 10
console.log(findFirstEven([1, 3, 5])); // null

File: python/stage2/S2_FOR_02.py
def find_first_even(nums):
    for n in nums:
        if n % 2 == 0:
            return n
    return None


# Tests
print(find_first_even([1, 3, 5, 6, 7]))
print(find_first_even([1, 3, 5]))
